docs/data-cookbooks/browse-generation.md

```diff
@@ -534,7 +534,7 @@ The processing lambda you construct will need to do the following:
 
 If you do not already have a CMR file in the granules list, you will need to generate one for valid export. This example's processing script generates and adds it to the `FilesToGranules` file list via the payload but it can be present in the InputGranules from the DiscoverGranules task as well if you'd prefer to pre-generate it.
 
-Both downstream tasks `MoveGranules` and `PostToCmr` expect a valid CMR file to be available if you want to export to CMR.
+Both downstream tasks `MoveGranules`, `UpdateGranulesCmrMetadataFileLinks`, and `PostToCmr` expect a valid CMR file to be available if you want to export to CMR.
```

Comment:
```suggestion
Both downstream tasks `UpdateGranulesCmrMetadataFileLinks`, and `PostToCmr` expect a valid CMR file to be available if you want to export to CMR.
```

Correct me if I'm wrong, but MoveGranules no longer does anything with CMR files with this PR?

Reply:
Hm, I don't think that is entirely true. MoveGranules still deals with CMR files in `updateGranulesMetadata()` https://github.com/nasa/cumulus/blob/master/tasks/move-granules/index.js#L79

Reply:
OK, then my nitpick is the the use of 'Both' for 3 tasks is slightly odd use of language, but not really a blocking issue.